release-team/role-handbooks/enhancements/README.md

```diff
@@ -101,48 +115,62 @@ It is important that this process be followed and documentation remain up-to-dat
 
 #### Post Enhancements Freeze (Week 5-10)
 
-- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, if there are PRs that have not been merged, move the issue to At Risk. If there is no activity, ping issue owners on either the issue or the k/k PR.
-- Monitor issues that are At Risk closely, almost daily. Code Freeze means no new code and keeping tabs on the status of the k/k PR is critical to planning. Make decisions if the enhancement should be deferred and work with SIG Leads to determine the best path forward.
+- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, if there are PRs that have not been merged, move the issue to `At Risk`. If there is no activity, ping issue owners on either the issue or the k/k PR.
+- Monitor issues that are `At Risk` closely, almost daily. Code Freeze means no new code and keeping tabs on the status of the k/k PR is critical to planning. Make decisions if the enhancement should be deferred and work with SIG Leads to determine the best path forward.
 
 #### Code Freeze (Week 10+)
 
-Start planning for the next release while assisting the Release Lead with anything relating to Analyst or Public Relation planning. Work with the Communications Lead to develop major themes for the official Kubernetes blog post.
+- Remove any enhancements that failed to merge their code by the Code freeze deadline.
+  - Set their status in the sheet to `Removed from Milestone`.
+  - Remove the milestone and change `tracked/yes` label to `tracked/no`.
+  - Remove the milestone from all open `k/k` PRs related to the enhancement.
+- Any enhancements removed from the milestone will now require an exception. As exception requests come in, discuss each with the Release Lead (and Shadows) to arrive at an approve/reject decision.
+  - Add a `/hold` label to all incoming exceptions to prevent from accidental merge.
+  - Add incoming exception information to the previous created `exception.yaml` file.
+  - If a previously removed Enhancement has had their exception Approved, set their status back to `Tracked`
+- Start planning for the next release while assisting the Release Lead with anything relating to Analyst or Public Relation planning. Work with the Communications Lead to develop major themes for the official Kubernetes blog post.
 
 
 ### Working with the Enhancement Tracking Sheet
 
 #### Cross Release Enhancement Tracking
 
-The source of truth for Enhancements is the `data` tab within the tracking sheet.
-It contains items that do not often change state. All other tabs are driven off
-the data in this tab. It should be kept as up-to-date as possible.
+The source of truth for Enhancements is the `data` tab within the tracking sheet. All other tabs are driven off the data in this tab.
```

Comment:
maybe note that certain updates should be made in this tab not the enhancements tab?

Reply:
Listed the columns that should be changed in the `data` tab

Reply:
also added to L124